http/src/main/resources/leo.conf

```diff
@@ -198,6 +198,7 @@ azure {
   cromwell-runner-app-config {
     instrumentation-enabled = ${?CROMWELL_RUNNER_APP_INSTRUMENTATION_ENABLED}
     enabled = ${?CROMWELL_RUNNER_APP_ENABLED}
+    ecm-base-uri = ${?ECM_URL}
```

Comment:
🤔 where will the URL come from for non BEE environments? It seems like you also need to specify it for the live environments in this PR https://github.com/broadinstitute/terra-helmfile/pull/5313/files

Reply:
That is a valid question and I should have linked another PR in here. Apologies for that! We did add the ecm-base-url for other environments in a previous PR - https://github.com/broadinstitute/terra-helmfile/pull/5236/files

Reply:
Ah perfect! In that case this looks good :) I will 👍